lib/material/internal/material_tabs.flow

```diff
@@ -52,6 +52,17 @@ MDynamicTabs2T(manager : MaterialManager, parent : MFocusGroup, m : MDynamicTabs
 		default : acc;
 	});
 
+	headerOnTop = \tabHeader, tabsContent ->
```

Comment:
It looks like it is worth being a general-purpose element.
There is already MLines2InvertZorder. Let's make a copy of it, name TLines2InvertZorder, and locate in 'tropic-gui.flow'